front-end/src/main/resources/static/index.html

```diff
@@ -69,15 +69,30 @@
   function renderApiMessages(accessToken) {
     // include the Bearer token in the request
     $.ajax({
-      url: resourceServerBaseUrl + "/api/messages",
-      headers: {
-        'Authorization': "Bearer " + accessToken
-      },
+        url: resourceServerBaseUrl + "/api/messages",
```

Comment:
Can you post a screenshot of how the errors would be rendered after this change?

Reply:
<img width="1179" alt="Screenshot 2020-12-22 at 19 41 09" src="https://user-images.githubusercontent.com/53248512/102917424-e18b1c80-448d-11eb-8138-493e53dcc691.png">

Reply:
<img width="1179" alt="Screenshot 2020-12-22 at 19 41 09" src="https://user-images.githubusercontent.com/71881713/102918771-3b8ce180-4490-11eb-9dfb-83a106ebee8a.png">

Reply:
Clearing the `404` console error for `GET /favicon.ico` is fine.

Apart from that IMHO, there is not much helpful information that is presented in the above screen that could assist the user in figuring out the issue. Not sure of the value this change would create for the user. Tell me if I'm missing anything here.

Reply:
I believe we should have at least some message to the user. Maybe something like that?
<img width="2001" alt="Screenshot 2020-12-29 at 11 54 38" src="https://user-images.githubusercontent.com/71881713/103275500-00793980-49cd-11eb-8beb-b42092424fdb.png">

Reply:
The latest screenshot above looks better actually. Though I like the idea of presenting the detailed error messages in browser console, a developer should still be able to use `Network` debugger in browser tools to figure out the request/response details in my opinion.